src/routes/github/github-router.ts

```diff
@@ -53,11 +54,11 @@ subRouter.post("/webhooks",
 
 // Create-branch is seperated above since it currently relies on query param to extract the jirahost
 // Todo able to move under the jirasymmetric middleware once flag completed
-subRouter.use("/create-branch", JiraHostFromQueryParamMiddleware, maybeJiraSymmetricJwtMiddleware, GithubServerAppMiddleware, GithubAuthMiddleware, csrfMiddleware, GithubCreateBranchRouter);
+subRouter.use("/create-branch", JiraHostFromQueryParamMiddleware, cookieSessionMiddleware, maybeJiraSymmetricJwtMiddleware, GithubServerAppMiddleware, csrfMiddleware, GithubCreateBranchRouter);
```

Comment:
I think `cookieSessionMiddleware` is provided outside above the github router for all routes, hence we don't need it here?

Reply:
NICE!